comixed-frontend/src/assets/i18n/fr/library.json

```diff
@@ -1,36 +1,36 @@
       },
       "button": {
-        "start": "Organize Library"
+        "start": "Organiser la bibliothèque"
       },
-      "warning-text": "Organizing your library cannot be stopped once started.",
+      "warning-text": "L'organisation de votre bibliothèque ne peut pas être arrêtée une fois commencée.",
       "effects": {
-        "success-detail": "Library consolidation has started...",
-        "error-detail": "Failed to start library consolidation..."
+        "success-detail": "La consolidation de la bibliothèque a commencé...",
+        "error-detail": "Échec de la consolidation de la bibliothèque..."
       },
-      "variables-note": "The following list shows the variables that can used in the renaming rules.",
+      "variables-note": "La liste suivante indique les variables qui peuvent être utilisées dans les règles de renommage.",
       "variables-example": "$PUBLISHER/$SERIES/$VOLUME/$SERIES v$VOLUME #$ISSUE ($COVERDATE)",
-      "variables-example-note": "This example organizes a library by publisher, series, volume and then a detailed filename. Note you do NOT need to specify the extension: that is added automatically.",
+      "variables-example-note": "Cet exemple organise une bibliothèque par éditeur, série, volume et ensuite un nom de fichier détaillé. Notez que vous n'avez pas besoin de spécifier l'extension : celle-ci est ajoutée automatiquement.",
       "variable": {
-        "PUBLISHER": "The name of the publisher of the comic",
-        "SERIES": "The name of the series of the comic",
-        "VOLUME": "The first year this volume of the comic was published",
-        "ISSUE": "The issue number for this isue",
-        "COVERDATE": "The coverdate for this issue"
+        "PUBLISHER": "Le nom de l'éditeur de la bande dessinée",
+        "SERIES": "Le nom de la série de la bande dessinée",
+        "VOLUME": "La première année de publication de ce volume de la bande dessinée",
+        "ISSUE": "Le numéro de la publication",
+        "COVERDATE": "La date de couverture pour cette publication"
       },
-      "delete-files-warning-header": "Delete Physical Files",
-      "delete-files-warning-message": "Deleting physical files cannot be undone. Are you SURE you want to do this?",
-      "confirmation-header": "Organize Library",
-      "confirmation-message": "Are you sure you want to {deletePhysicalFiles, select, true{delete comic files marked for deletion and} other{}} organizing the library?"
+      "delete-files-warning-header": "Suppression des fichiers physiques",
+      "delete-files-warning-message": "La suppression des fichiers physiques ne peut pas être annulée. Êtes-vous sûr de vouloir le faire ?",
+      "confirmation-header": "Organiser la bibliothèque",
+      "confirmation-message": "Êtes-vous sûr de vouloir {effacer les fichiers physiques, sélectionner, true {effacer les fichiers dex bande dessinée marqués pour suppression et} autres { et}} organiser la bibliothèque ?"
```

Comment:
The variable "deletePhysicalFiles" was translated and needs to be reverted.

The keyword "other" was translated here and needs to be reverted. Also there shouldn't be a space between the keyword and the opening curly brace.

Reply:
fixed